--- docs/conf/inspircd.conf.example.orig	2018-01-17 18:57:56.868467112 -0700
+++ docs/conf/inspircd.conf.example	2018-01-19 14:59:20.417097893 -0700
@@ -152,7 +152,7 @@
       # to either "gnutls" or "openssl". The appropriate SSL module must be
       # loaded for SSL to work. If you do not want the port(s) in this bind
       # tag to support SSL, just remove or comment out this option.
-      ssl="gnutls"
+      ssl="openssl"
 >
 
 <bind address="" port="6660-6669" type="clients">
@@ -421,11 +421,11 @@
 
 # This file has all the information about oper classes, types and o:lines.
 # You *MUST* edit it.
-<include file="conf/examples/opers.conf.example">
+<include file="examples/opers.conf.example">
 
 # This file has all the information about server links and ulined servers.
 # You *MUST* edit it if you intend to link servers.
-<include file="conf/examples/links.conf.example">
+<include file="examples/links.conf.example">
 
 #-#-#-#-#-#-#-#-#-#-  MISCELLANEOUS CONFIGURATION  -#-#-#-#-#-#-#-#-#-#
 #                                                                     #
@@ -960,7 +960,7 @@
 # provide almost all the features of InspIRCd. :)                     #
 #                                                                     #
 # The default does nothing -- we include it for simplicity for you.   #
-<include file="conf/examples/modules.conf.example">
+<include file="examples/modules.conf.example">
 
 # Here are some pre-built modules.conf files that closely match the
 # default configurations of some popular IRCd's. You still may want to
@@ -972,10 +972,10 @@
 # recommended that you make your own modules file based on modules.conf.example.
 
 # Settings similar to UnrealIRCd defaults.
-#<include file="conf/examples/modules/unrealircd.conf.example">
+#<include file="examples/modules/unrealircd.conf.example">
 
 # Settings similar to Charybdis IRCd defaults.
-#<include file="conf/examples/modules/charybdis.conf.example">
+#<include file="examples/modules/charybdis.conf.example">
 
 
 #########################################################################
